drivers/serial/uart_mcux_flexcomm.c

```diff
@@ -1132,7 +1133,7 @@ static int mcux_flexcomm_pm_action(const struct device *dev, enum pm_device_acti
 	case PM_DEVICE_ACTION_TURN_OFF:
 		break;
 	case PM_DEVICE_ACTION_TURN_ON:
-		int ret = mcux_flexcomm_init_common(dev);
+		ret = mcux_flexcomm_init_common(dev);
 
```

Comment:
can you drop this blank line while you are at it? Normally ret check are just after ret assignment, which makes sense since you can see the whole block as a logical: do something and check the result